README.md

```diff
@@ -14,9 +14,7 @@ The streams start **September 3rd** and end **September 13th**. Hack submissions
 
 ## Register
 
-Register for the hackathon on the [Reactor "RAG 101" session page](https://aka.ms/raghack/register). This will register you for both the first stream and the hackathon.
-
-Visit the [Reactor series home page](https://reactor.microsoft.com/reactor/series/s-1369/) to see all sessions!
+Register for the hackathon using any of the sessions linked on the [Reactor series home page](https://reactor.microsoft.com/en-us/reactor/series/s-1369/). This will register you for both the selected session and the hackathon.
```

Comment:
```suggestion
Register for the hackathon using any of the sessions linked on the [Reactor series home page](https://reactor.microsoft.com/reactor/series/s-1369/). This will register you for both the selected session and the hackathon.
```